airflow/providers/amazon/aws/transfers/sql_to_s3.py

```diff
@@ -134,11 +127,23 @@ def __init__(
             raise AirflowException(f"The argument file_format doesn't support {file_format} value.")
 
     @staticmethod
-    def _fix_dtypes(df: pd.DataFrame, file_format: FILE_FORMAT) -> None:
```

Comment:
You don't need to remove this annotation. 
Just add import in existed `if TYPE_CHECKING` block, everything inside it this only evaluates on static check (mypy, etc.) and IDEs (PyCharm, VSCode, etc.) but not in runtime.

```python
if TYPE_CHECKING:
    from airflow.utils.context import Context

```

Reply:
Done, I've moved only the `DataFrame` import up to the `if TYPE_CHECKING ` since the other imports `Int64Dtype` and `Float64Dtype` are required for type casting later in the `_fix_dtypes` function